modules/farm_fd2_school/src/entrypoints/FD4_Application/harvest.e2e.cy.js

```diff
@@ -12,7 +12,58 @@ describe('Tests for the Harvest form', () => {
     cy.saveSessionStorage();
   });
 
-  it('Placeholder test', () => {
-    cy.get('[data-cy="does-not-exist"]');
+  it('Initial state of the Harvest form is correct', () => {
+    cy.get('[data-cy="harvest-header"]')
+      .should('be.visible')
+      .and('contain', 'Harvest');
+
+    cy.get('[data-cy="harvest-date"]')
+      .should('be.visible')
+      .find('input')
+      .should('have.value', '2019-06-15');
+
+    cy.get('[data-cy="harvest-crop"]')
+      .should('be.visible')
+      .find('select')
+      .should('have.value', null);
+
+    cy.get('[data-cy="harvest-buttons"]').should('be.visible');
+    cy.get('[data-cy="harvest-buttons"]')
+      .find('button')
```

Comment:
It would be better to use the `data-cy` for the "Submit" button here.  See the documentation for the `SubmitResetButtons` component.